test/src/main/configurations/HTTP/ConfigurationRestListener.xml

```diff
@@ -191,7 +191,7 @@
 			<pipe name="callRest"
 				className="nl.nn.adapterframework.pipes.SenderPipe">
 				<sender className="nl.nn.adapterframework.http.HttpSender"
-					urlParam="url_param" base64="true">
```

Comment:
Gaat dit zomaar goed? Nu zet de HttpSender het resultaat niet om in een base64 maar hij gaat nog steeds naar de base64 pipe?

Reply:
de forward van pipe name="callRest" gaat naar een nl.nn.adapterframework.pipes.Base64Pipe